README.md

```diff
@@ -1,39 +1,597 @@
+    #! Usage from Daru::DataFrame
+    df.to_csv(args)
+    ```
+
+### Excel Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.xls** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/Excel)
+- **Gem Dependencies**: `spreadsheet` gem
+- **Other Dependencies**: None
+- **Usage**:
+    ```ruby
+    #! Partially require just Excel Exporter
+    require 'daru/io/exporters/excel'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::Excel.new(df, args).call
+
+    #! Usage from Daru::DataFrame
+    df.to_excel(args)
+    ```
+
+### JSON Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.json** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/JSON)
+- **Gem Dependencies**: Standard library `json` gem and `jsonpath` gem
+- **Other Dependencies**: None
+- **Usage**:
+    ```ruby
+    #! Partially require just JSON Exporter
+    require 'daru/io/exporters/json'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::JSON.new(df, args).call
+
+    #! Usage from Daru::DataFrame
+    df.to_json(args)
+    ```
+
+### RData Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports multiple **Daru::DataFrame**s into a **.rdata** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RData)
+- **Gem Dependencies**: `rsruby` gem
+- **Other Dependencies**: R and setting of `R_HOME` variable
+- **Usage**:
+    ```ruby
+    #! Partially require just RData Exporter
+    require 'daru/io/exporters/r_data'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::RData.new(args).call
+    ```
+
+### RDS Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.rds** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RDS)
+- **Gem Dependencies**: `rsruby` gem
+- **Other Dependencies**: R and setting of `R_HOME` variable
+- **Usage**:
+    ```ruby
+    #! Partially require just RDS Exporter
+    require 'daru/io/exporters/rds'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::RDS.new(df, args).call
+
+    #! Usage from Daru::DataFrame
+    df.to_rds(args)
+    ```
+
+### SQL Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into an SQL table.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/SQL)
+- **Gem Dependencies**: `dbd-sqlite3`, `dbi` and `sqlite3` gems
+- **Other Dependencies**: SQL
+- **Usage**:
+    ```ruby
+    #! Partially require just SQL Exporter
+    require 'daru/io/exporters/sql'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::SQL.new(df, args).call
+
+    #! Usage from Daru::DataFrame
+    df.to_sql(args)
+    ```
+
+# Tweaking IO modules
```

Comment:
I don't believe "Tweaking" is a good word here. Let's do a section named "Creating your own importers/exporters"